### DISCUSSION TITLE
润学伦理与国际主义精神

### DISCUSSION THREAD

**Discussion by Andrew1771-re** [The-Run-Philosophy-Organization/run#208] [General]
润是一种选择，一种行动，更是一种精神。大音希声，大象无形，大润不移。何谓大润？不是「身」润，是「心」润。孟母三迁，是小润；孔子周游，是中润；马列出国，是大润。马克思不润，写不出资本论。马克思为什么润？是因为有了润的精神。一个人，毫无利己的动机，把世界人民的解放事业当作他自己的事业，这是什么精神？这是国际主义的精神，这是共产主义的精神，这就是润的精神。每个中国人都应该去读一读卡尔·亨利希·马克思发布的中学毕业论文《青年在选择润时的考虑》。我将其引述至此。

自然本身给动物规定了它应该遵循的活动范围，动物也就安分地在这个范围内活动，不试图越出这个范围，甚至不考虑有其他什么范围的存在。神也给人指定了共同的目标──使人类和他自己趋于高尚，但是，神要人自己去寻找可以达到这个目标的手段──润；神让人在社会上选择一个最适合于他、最能使他和社会都得到提高的地位。

能有润的选择是人比其他生物远为优越的地方，但是这同时也是可能毁灭人的一生、破坏他的一切计划并使他陷于不幸的行为。因此，认真地考虑自己该不该润──这无疑是开始走上生活道路而又不愿拿自己最重要的事业去碰运气的青年的首要责任。

每个人眼前都有一个目标，这个目标至少在他本人看来是伟大的，而且如果最深刻的信念，即内心深处的声音，认为这个目标是伟大的，那他实际上也是伟大的，因为神决不会使世人完全没有引导；神总是轻声而坚定地作启示。

但是，这声音很容易被淹没；我们润的念头可能须臾而生，同样可能须臾而逝。也许，我们的幻想由然而生，我们的感情激动起来，我们的眼前浮想联翩，我们狂热地追求我们以为是神本身给我们指出的目标；但是，我们梦寐以求的东西很快就使我们厌恶──于是我们的整个存在也就毁灭了。

因此，我们应当认真考虑：润，是不是真正使我们受到鼓舞？我们的内心是不是同意？我们受到的鼓舞是不是一种迷误？我们认为是神的召唤的东西是不是一种自欺？但是，不找出鼓舞的来源本身，我们怎么能认清这些呢？

润是光辉的，光辉则引起虚荣心，而虚荣心容易给人鼓舞或者是一种我们觉得是鼓舞的东西；但是，被名利弄得鬼迷心窍的人，理智已无法支配他，于是他一头栽进那不可抗拒的欲念驱使他去的地方；他已经不再自己选择他在社会上的地位，而听任偶然机会和幻想去决定它。

润学的使命绝不是移民去一个最足以炫耀的国度，因为那将不是一个使我们长期生活而始终不会感到厌倦、始终不会松动、始终不会情绪低落的国度，相反，我们很快就会觉得，我们的愿望没有得到满足，我们理想没有实现，我们就将怨天尤人。

但是，不只是虚荣心能够引起对这种或那种国度突然的热情。也许，我们自己也会用幻想把这些国家美化，把它美化成人生所能提供的至高无上的东西。我们没有仔细分析它，没有衡量它的全部份量，即它让我们承担的重大责任；我们只是从远处观察它，然而从远处观察是靠不住的。

在这里，我们自己的理智不能给我们充当顾问，因为它既不是依靠经验，也不是依靠深入的观察，而是被感情欺骗，受幻想蒙蔽。然而，我们的目光应该投向哪里呢？在我们丧失理智的地方，谁来支持我们呢？

是我们的父母， 他们走过了漫长的生活道路， 饱尝了人世的辛酸。──我们的心这样提醒我们。

如果我们通过冷静的研究，认清润学的全部份量，了解它的困难以后，我们仍然对它充满热情，我们仍然爱它。觉得自己适合它，那时我们就应该选择它，那时我们既不会受热情的欺骗，也不会仓促从事。

但是，我们并不能总是能够选择我们自认为适合的道路；我们在社会上的关系，还在我们有能力对它们起决定性影响以前就已经在某种程度上开始确立了。现实中阻碍润的困难常常威胁我们，任何人都不敢忽视这些困难。诚然，我们能够努力工作攒钱，但这么一来，我们也就垮得更快；在这种情况下，我们就是冒险把大厦筑在松软的废墟上，我们的一生也就变成一场精神原则和肉体原则之间的不幸的斗争。但是，一个不能克服自身相互斗争的因素的人，又怎能抗拒生活的猛烈冲击，怎能安静地从事润的活动呢？只有从安静中才能产生伟大壮丽的事业，安静是唯一生长出「润之果实」的土壤。

尽管由于现实的阻挠不适合我们去润，不能持久地工作，而且工作起来也很少乐趣，但是，为了润而牺牲自己幸福的思想激励着我们不顾体弱去努力工作。如果我们选择了超出自己能力去润的国家，那么我们很快就会自愧无能，并对自己说，我们是无用的人，是不能完成自己使命的社会成员。由此产生的必然结果就是妄自菲薄。还有比这更痛苦的感情吗？还有比这更难于靠外界的赐予来补偿的感情吗？妄自菲薄是一条毒蛇，它永远啮噬着我们心灵，吮吸着其中滋润生命的血液，注入厌世和绝望的毒液。

如果我们错误地估计了自己的能力，以为能够润去经过周密考虑而选定的国家，那么这种错误将使我们受到惩罚。即使不受到外界指责，我们也会感到比外界指责更为可怕的痛苦。

如果我们把这一切都考虑过了，如果我们生活的条件容许我们选择任何一个国家；那么我们就可以选择一个能使我们最有尊严的国家；选择一种建立在我们深信其正确的思想上的国家；选择一种能给我们提供广阔场所来为人类进行活动、接近共同目标（对于这个目标来说，一切职业只不过是手段）即完美境地的国家。

尊严就是最能使人高尚起来、使他的活动和他的一切努力具有崇高品质的东西，就是使他无可非议、受到众人钦佩并高出于众人之上的东西。

但是，能给人以尊严的只有这样的国家，在这样的国家我们不是作为奴隶般的工具，而是在自己的领域内独立地进行创造；不需要有不体面的行动（哪怕只是表面上不体面的行动），甚至最优秀的人物也会怀着崇高的自豪感去润到那里。最合乎这些要求的国家，并不一定是最富裕强大的国家，但总是最可取的国家。

但是，正如令人有失尊严的国家会贬低我们一样，那种建立在我们后来认为是错误的思想上的国家也一定使我们感到压抑。这里，我们除了自我欺骗，别无解救办法，而以自我欺骗来解救又是多么糟糕！对于那些尚未踏入社会，而是从事抽象真理研究的学生；对于还没有坚定的原则和牢固、不可动摇的信念的青年是最危险的。同时，如果润学在我们心里深深地扎下了根，如果我们能够为它们的支配思想牺牲生命、竭尽全力，这些看来似乎还是最高尚的。

润学能够使才能适合的人幸福，但也必定使那些不经考虑、凭一时冲动就仓促从事的人毁灭。相反，重视作为我们职业的基础的润学思想，会使我们在社会上占有较高的地位，提高我们本身的尊严，使我们的行为不可动摇。

一个选择了自己所珍视的国度的人，一想到他可能不称职时就会战战兢兢──这种人单是因为他在社会上所居地位是高尚的，他也就会使自己的行为保持高尚。

在选择国家时，我们应该遵循的原则主要是人的幸福和我们自身的完美。不应认为，这两种利益是敌对的，互相冲突的，一种利益必须消灭另一种的；人类的天性本来就是这样的：人们只有为同时代人的完美、为他们的幸福而工作，才能使自己也达到完美。

如果一个人只为自己而润，他也许能够成为著名的学者、大哲人、卓越诗人、小有积蓄的程序员，然而他永远不能成为完美无疵的伟大人物。

历史承认那些为共同目标劳动因而自己变得高尚的人是伟大人物；经验赞美那些为大多数人带来幸福的人是最幸福的人；宗教本身也教诲我们，人人敬仰的理想人物，耶稣就曾为人类牺牲了自己──有谁敢否定这类教诲呢？

如果我们选择了最能为人类福利考虑而产生的润学，那么，重担就不能把我们压倒，因为这是为大家而献身；那时我们所感到的就不是可怜的、有限的、自私的乐趣，我们的幸福将属于千百万人，我们的事业将默默地、但是永恒发挥作用地存在下去，面对我们的骨灰，高尚的人们将洒下热泪。

**Comment by the-run-philosophy** [comment]
我收录进感悟里
